site-src/v1alpha2/references/policy-attachment.md

```diff
@@ -0,0 +1,294 @@
+# Policy Attachment
+
+Policies attached to Gateway API resources and implementations should use the
+following approach to ensure consistency across implementations of the API.
+There are three primary components of this pattern:
+
+* A shared means of attaching policy to resources.
+* Support for configuring both default and override values within policy
+  resources.
+* A shared hierarchy to illustrate how default and override values should
+  interact.
+
+This kind of standardization will not only enable consistent patterns, it will
+enable tooling such as kubectl plugins to be able to visualize all policies that
+have been applied to a given resource.
+
+## Policy Attachment for Ingress
+Attaching policy to Gateway resources for ingress use cases is relatively
+straightforward. A policy can reference the resource it wants to apply to.
+Access is granted with RBAC - anyone that has access to create a RetryPolicy in
+a given namespace can attach it to any resource within that namespace.
+
+![Simple Ingress Example](/geps/images/713-ingress-simple.png)
+
+To build on that example, it’s possible to attach policies to more resources.
+Each policy applies to the referenced resource and everything below it in terms
+of hierarchy. Although this example is likely more complex than many real world
+use cases, it helps demonstrate how policy attachment can work across
+namespaces.
+
+![Complex Ingress Example](/geps/images/713-ingress-complex.png)
```

Comment:
404

Reply:
404 for all pngs

Reply:
I'd also recommending separating GEP content from site content. No Big Deal if later.

Reply:
Good call, moved all images to a separate /images/policy directory.